hodometer/Dockerfile.hodometer

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20.11-bullseye as builder
+FROM golang:1.20.13-bullseye as builder
```

Comment:
Just wondering out loud here, whether it would make sense to remove very precise tags like these from Dockerfiles, similarly to how you've changed things in the github workflows from 1.20.11 to just 1.20. The equivalent image here would be `golang:1.20-bullseye`. 

This would give us some upgrades for free when rebuilding images, but it does add some risk that things break. In general I think go is better than others in maintaining compatibility, but it's a discussion worth having (whether to take on that risk). 

However, irrespective of the choice, I would apply the same strategy both here and in the github actions to maintain exact compatibility between the tests that github runs and the docker images being built.

Reply:
For now I pinned it to the same versions. Lets discuss if we can relax it as suggested later. Hopefully this will all be done by dependabot.
I did relax it in github workflow initially but I was not sure completely if they are available in lock steps.

Reply:
as discussed we will relax go patch version in docker images.